### ISSUE TITLE
Transpiler downloaded unnecessarily.

### ISSUE THREAD

**Issue by Zoltu** [systemjs/systemjs#1250]
I have an application that has been bundled with `jspm bundle`.  If I look at the bundle I can see that it has transpiled all of my TypeScript code into JS and put it into the bundle.

However, my application still attempts to download the typescript transpiler, even though it never actually downloads any TypeScript.

I am not sure what Aurelia is doing to trigger this but I was unable to cause it by manually calling `System.import('source/app.ts')`.

Repro steps
====

1. `mkdir test`
1. `cd test`
1. `npm install jspm@0.17.0-beta.14`
1. `.\node_modules\.bin\jspm init`
 * Quick with defaults except for transpiler `TypeScript` and package name `foo`
1. `.\node_modules\.bin\jspm install aurelia-bootstrapper`
1. add a `map` entry to `jspm.config.js`:
 * `"app": "src/app.ts"`
1. Change the meta `"*.js"` to `"*.ts"` in `jspm.config.js`
1. Create `src/app.html` with:

  ```html
  <template>
  	<h1>${content}</h1>
  </template>
  ```
1. Create `src/app.ts` with:

	```ts
	export class App {
		content: String;
	
		constructor() {
			this.content = "Hello World!";
		}
	}
	```
1. Create `index.html` with:

	```html
	<!doctype html>
	<html>
		<body aurelia-app>
			<div style="text-align: center; font-size:300%">Loading...</div>
	
			<script src="jspm_packages/system.src.js"></script>
			<script src="jspm.browser.js"></script>
			<script src="jspm.config.js"></script>
			<script src="build.js"></script>
			<script>
				System.import('aurelia-bootstrapper');
			</script>
		</body>
	</html>
	```
1. `.\node_modules\.bin\jspm bundle src\**\*.ts`
1. `npm install node-static`
1. `.\node_modules\.bin\static .`

**Comment by guybedford** [comment]
I was able to follow the steps here but I wasn't seeing the transpiler downloaded when using the bundle. Are you sure you're not using other custom configuration that may be interfering with the bundle loading?

**Comment by Zoltu** [comment]
I don't believe so, as I wrote up these steps I was executing them at the same time to make sure I didn't miss anything.  I'll go through them again this evening after having forgotten what I was doing to see if I can reproduce it again.  Are there any hidden settings that JSPM may be using that could cause this?  I am seeing the same problem on my build server (AppVeyor), suggesting it isn't an environment issue.

If I can reproduce it again I'll put up a git repo with the problem as well to further help troubleshooting.

**Comment by Zoltu** [comment]
It appears I missed a critical step in my instructions!

Add `"format": "esm",` to `jspm.config.js` the `foo` package like so:
```js
SystemJS.config({
  transpiler: "plugin-typescript",
  packages: {
    "foo": {
      "format": "esm",
      ...
    }
    ...
  }
  ...
});
```

**Comment by Zoltu** [comment]
Reproduction case repository: https://github.com/Zoltu/jspm-test

**Comment by guybedford** [comment]
Thanks, yes this is a SystemJS bug that it is trying to treat the `app.html` file as an ES module. I've added a fix in https://github.com/systemjs/systemjs/commit/4bbfdcf8da782534b5e8236d75e53a8730d4a49d here.

**Comment by guybedford** [comment]
Released in 0.19.28.
